src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/DecisionOutcomeCallbackHandler.java

```diff
@@ -39,9 +42,11 @@ protected Map<String, Callback> callbacks() {
 
     private CallbackResponse changeState(CallbackParams callbackParams) {
         if (featureToggleService.isCaseProgressionEnabled()) {
+            String authToken = callbackParams.getParams().get(BEARER_TOKEN).toString();
             CaseData caseData = callbackParams.getCaseData().toBuilder()
                 .build();
             CaseData.CaseDataBuilder<?, ?> caseDataBuilder = caseData.toBuilder();
+            deleteNotificationAndInactiveTasks(caseData, authToken);
```

Comment:
Should we not be doing this in the dashboard handlers:
uk/gov/hmcts/reform/civil/handler/callback/camunda/dashboardnotifications/defendant/DefendantDecisionOutcomeDashboardHandler.java
uk/gov/hmcts/reform/civil/handler/callback/camunda/dashboardnotifications/claimant/ClaimantDecisionOutcomeDashboardHandler.java

Reply:
@m-meulendijks-v1 I have updated, please have a look. Thanks.